### DISCUSSION TITLE
Moes 2 Gang Dimmer Module not dimming fully

### DISCUSSION THREAD

**Discussion by elngejw** [arendst/Tasmota#19933] [Support request]
Hi all,

I have a Tasmota flashed Moes 2 Gang Dimmer Module (MS-105B), working fine except I can’t get it configured to dim fully. Lowest dimming keeps leds still lighted at let’s say 25%. Tried some other DimmerRange values, but no success for dimming it fully.

Didn’t find any other possible options to try. Does anybody have an idea where to look for other options to try?

Moes 2 Gang Dimmer Module (MS-105B) is refurbished with a 8266 ESP-01S because this one I had available instead of recommended ESP-02S . I’m not sure this could create issues.

Many thanks for your support!

Grtz, Geert-Jan

**Comment by sfromis** [comment]
When getting a device with a [TuyaMCU](https://tasmota.github.io/docs/TuyaMCU/) infestation, Tasmota is not in control of device operation, except by trying to act as a "remote control" to the main chip where all the details are handled. This may mean that you basically are out of luck for wanting to tweak. I've carefully avoided getting into TuyaMCU stuff, but in some cases the tool [Tasmota-Tuya-Helper](https://github.com/sillyfrog/Tasmota-Tuya-Helper) can be useful for seeing what's available.

Whatever ESP8266 module you use should make no difference, as long as it works.
